src/content/translations/it/developers/tutorials/uniswap-v2-contract-walkthrough/index.md

```diff
@@ -0,0 +1,1962 @@
+---
+title: "Guida dettagliata al contratto Uniswap-v2"
```

Comment:
This file name is incorrect and needs to be changed in Crowdin: https://ethereumorgwebsitedev01-ittutorials.gtsb.io/it/developers/tutorials/uniswap-v2-annotated-code/

`uniswap-v2-annotated-code` is the correct path